docs/root/configuration/listeners/network_filters/client_ssl_auth_filter.rst

```diff
@@ -22,7 +22,7 @@ Every configured client TLS authentication filter has statistics rooted at
   update_success, Counter, Total principal update successes
   update_failure, Counter, Total principal update failures
   auth_no_ssl, Counter, Total connections ignored due to no TLS
-  auth_ip_white_list, Counter, Total connections allowed due to the IP white list
+  auth_ip_allowlist, Counter, Total connections allowed due to the IP allowlist
```

Comment:
I went and just outright changed this stat name as a) this filter is not heavily used and b) this is not a super critical stat so I think people can deal with this change. Happy to not do this if people feel strongly about it.

Reply:
I don't feel strongly about this, but independent of this, do we have a stats deprecation policy? This came up recently in https://github.com/envoyproxy/envoy/pull/11697#commitcomment-40148702

Reply:
I opened https://github.com/envoyproxy/envoy/issues/11767 and will merge this.